### PR TITLE
Natss install

### PR DESCRIPTION
**Pull request by matzew**
BASED on: #2194

Changes:
* Adding content for NATSS Channel install

/assign @Abd4llA  please review

### CODE REVIEW

File: docs/install/knative-with-any-k8s.md [hunk 1]
```diff
@@ -380,8 +380,19 @@ To learn more about the Apache Kafka channel, try [our sample](../eventing/sampl
 
 {{< /tab >}}
 
+{{% tab name="NATS Channel" %}}
+
+1. First, [Install NATS Streaming for Kubernetes](https://github.com/knative/eventing-contrib/blob/{{< version >}}/natss/config/broker/README.md)
+
+1. Then install the Nats Streaming Channel:
```
Comment (antoineco) [inline docs/install/knative-with-any-k8s.md]: "NATS" (capitalized)

**Comment by vaikas** [comment]
Sweet!
/lgtm

**Pull request merged**